app/scripts/modules/core/src/pipeline/config/stages/managed/ImportDeliveryConfigStageConfig.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { SETTINGS } from 'core/config/settings';
+import { IStageConfigProps, FormikStageConfig } from 'core/pipeline';
+import { FormikFormField, TextInput } from 'core/presentation';
+import { HelpField } from 'core/help';
+
+export const ImportDeliveryConfigStageConfig: React.SFC<IStageConfigProps> = stageConfigProps => (
+  <FormikStageConfig
+    {...stageConfigProps}
+    onChange={stageConfigProps.updateStage}
+    render={() => (
+      <div className="form-horizontal">
+        <FormikFormField
+          name="directory"
+          label="Directory"
+          help={<HelpField id="pipeline.config.deliveryConfig.directory" />}
+          input={props => <TextInput {...props} placeholder={'/'} />}
```

Comment:
Shouldn't this use `SETTINGS.managedDelivery?.manifestBasePath` as its placeholder?

Reply:
Another piece I struggled with. After discussing with @emjburns, we agreed to restrict the manifest look up code in igor to look under a configurable "base path" (which is what this setting represents), possibly under additional subdirectories. So, for instance, if your base path is `.spinnaker`, your directory is `managed`, and your manifest is `myapp.yml`, then the full path in the SCM repo would be `.spinnaker/managed/myapp.yml`.

I thought of using the base path as a placeholder here, but was afraid that might lead people to actually type in the path _including_ the base path (e.g. `.spinnaker/managed`), which would not actually work. The model for this turned up a bit wonky, I think... 😞 If you have any other ideas on how to make this better, please let me know.

Reply:
er. yeah. That's all really confusing and wasn't in any way apparent or clear to me after reading documentation + code + seeing screenshots. Need to noodle a little more on what you just said before I decide if I have any other ideas.

Reply:
P.S. Maybe we could just prefix the text field with the base path? Is there a form field control that allows me to have a non-editable string (other than the label)?

Reply:
clarification on this as I'm thinking through it: do we have an actual real world example where we want to support a custom subdirectory beyond the one defined at the spinnaker installation level (e.g. your example of `.spinnaker/managed/xyz.yml`)? I am finding myself skeptical that such a setting is useful — originally I had thought the `manifestBasePath` setting was a _default_ and by setting this field you'd be changing the `.spinnaker` to something else. Knowing what it actually does, I have very little confidence it'd be useful — perhaps that's too naive an assumption.

Reply:
FWIW, I tried to explain that in the help bubble:
![image](https://user-images.githubusercontent.com/1323478/71859281-311ce080-30a3-11ea-9fbf-089600e49805.png)

Reply:
> clarification on this as I'm thinking through it: do we have an actual real world example where we want to support a custom subdirectory beyond the one defined at the spinnaker installation level (e.g. your example of `.spinnaker/managed/xyz.yml`)? I am finding myself skeptical that such a setting is useful — originally I had thought the `manifestBasePath` setting was a _default_ and by setting this field you'd be changing the `.spinnaker` to something else. Knowing what it actually does, I have very little confidence it'd be useful — perhaps that's too naive an assumption.

I have to agree. I feels like premature optimization. I forget why I even went down this road... @emjburns Any objections to removing this field from the form?

Reply:
Originally you could specify a directory because we had lots of resources, and you might not want them polluting your base folder (the rocket team does this). For our testing purposes, removing this directory would make it impossible to have a prestaging and a main manifest file unless we had two separate repositories. I'd prefer to keep this functionality for easy testing. Would it be easier to have the field called `Path` and pre-fill it with `.spinnaker/spinnaker.yml`? That way the behavior is clear, and the defaults should work for most people.

Reply:
I was thinking the same thing about changing this to a `Path` field, but just to play devil's advocate, in your example couldn't you just have a manifest called `spinnaker-main.yml` and another `spinnaker-staging.yml` in the same directory?

Reply:
Ah, true. If we don't require the name to be `spinnaker.yml` then there are a lot more possibilities. 

I'm fine with whatever you choose as long as there is a way to have two different manifests in a directory and only submit one! My mind is confusing the requirements of the hacky docker script with this new and improved flow.

Reply:
I had an in-person chat w/ Emily earlier, and based on the prestaging/main use case described I think my personal view has shifted to being fine with either:

- Having a static visual treatment sitting to the left of the directly text field (but connected to it) that has whatever your base path is. So `.spinnaker/` in a little static thing, and then the text field after it (this is basically Luis's earlier suggestion/idea)
- Taking the directory config out entirely and just having a single file name field where you can do `spinnaker-prestaging.yml`/`spinnaker-main.yml` etc. We'd still want to show the base path, but we'd keep it read-only

Just my 2 cents on paths forward, whatever you decide is great.

Reply:
Thanks @erikmunson. For that first idea, is there an existing component in deck that provides the little static  thing to the left of a text field? 😃

Reply:
sadly I don't know of any existing components that do this — if you decide to go that route we'll likely need to whip up something ourselves.